src/pages/dashboard.vue

```diff
@@ -174,10 +174,94 @@ const pendingAddress = computed(() => {
 
 const isChangeAddressDisabled = computed(() => business.currentBusiness.adminFreeze || pendingAddress.value)
 const isChangeDirectorDisabled = computed(() => business.currentBusiness.adminFreeze)
+const showChangeOfAddress = ref(false)
+
+const toggleChangeOfAddress = (show: boolean) => {
```

Comment:
just a note for future, toggle usually switches states if it was off -> on; false -> true.
Here, name would be maybe bit more self documenting if you would name method `isShowChangeOfAddress`

Reply:
is to met would return the value so I've changed it to be setChangeOfAddress